apps/arena/lib/arena/configuration.ex

```diff
@@ -77,53 +82,58 @@ defmodule Arena.Configuration do
     end)
   end
 
-  defp parse_mechanic_config(mechanic) when map_size(mechanic) > 1 do
-    raise "Config has mechanic with 2 values: #{inspect(mechanic)}"
-  end
+  # defp parse_mechanic_config(mechanic) when map_size(mechanic) > 1 do
```

Comment:
If we're not using it we should delete it

Reply:
Done!